### PR TITLE
ESY-3038 - Added documentation for new Trustly Direct Debit feature (…

### PR DESCRIPTION
**Pull request by Eddkus**
…Recur and unscheduled)

### CODE REVIEW

**Review by arebra — CHANGES_REQUESTED** [review]
Looks good, if we fix the small thing with consumer --> payer

File: _includes/unscheduled-purchase-trustly.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+is a payment which uses a `paymentToken` generated through a previous payment in
+order to charge the same card at a later time. They are done by the merchant
+without the user being present.
+
+To use this, you need to make sure `directDebitEnabled` is `true` in the Trustly contract. After that you need to have the field `generateUnscheduledToken` set to `true` when creating an initial Purchase or Verify. Another important step is to make sure the `email` field is set, as this is a required parameter.
+
+{:.code-view-header}
+**Request**
+
+```http
```
Comment (sigridbra) [inline _includes/unscheduled-purchase-trustly.md:15]: For me, it feels weird to document this as this is never relevant to the merchant

Comment (Eddkus) [reply _includes/unscheduled-purchase-trustly.md:15]: Do you suggest removing the request from the documentation?

File: _includes/recurring-trustly-payments.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+## Trustly recurring
+
+{% include jumbotron.html body="A service where consumer interaction is not possible, where the payment is pulled from the consumer’s bank account by sending a request to Trustly. " %}
```
[the remark this answers is not in the document]
Comment (Eddkus) [reply _includes/recurring-trustly-payments.md]: Noted 👍🏻

**Review by arebra — APPROVED** [review]
Great :)

File: _includes/unscheduled-purchase-trustly.md (hunk 1 above)
Comment (arebra) [reply _includes/unscheduled-purchase-trustly.md]: Is this resolved? :)

Comment (Eddkus) [reply _includes/unscheduled-purchase-trustly.md]: Yes, agreed that it wasn't relevant info for the merchant, so removed the request-response objects

File: payment-instruments/trustly/features/optional/recur.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+estimated_read: 5
+description: |
+  Setting up subscriptions and recurring payments.
+menu_order: 1500
+icon:
+  content: cached
+  outlined: true
+---
+
+{% include recurring-trustly-payments.md}
```
Comment (asbjornu) [inline payment-instruments/trustly/features/optional/recur.md:12]:
```suggestion
{% include recurring-trustly-payments.md %}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Pull request merged**